src/style.css

```diff
@@ -0,0 +1,121 @@
+:root {
+  --card-bg-color: #fff;
+  --card-brdr-color: #f3f3f3;
+  --button-main-color: #00acdc;
+  --button-text-color: #fff;
+  --button-hover-bg-color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+}
+
+body {
+  margin: 0;
+}
+
+h1 {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 14px;
+  border: 1px solid var(--card-brdr-color);
+  border-radius: 5px;
+  background-color: var(--card-bg-color);
+}
+
+.card__photo {
+  margin: 16px 0 42px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__description {
+  font-size: 12px;
+  line-height: 15px;
+  font-weight: 500;
+}
+
+.card__code {
+  width: 100%;
+  text-align: left;
+  margin: 5px 0 17px;
+}
+
+.card__rating,
+.card__price {
+  display: inherit;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.stars {
+  height: 15px;
+}
+
+.stars__item {
+  display: inline-block;
+  content: url(./images/star.svg);
+  margin: 0 2px 0 1px;
+  width: 13.3px;
+  height: 12px;
+}
+
+.stars--4 :nth-child(-n+4) {
```

Comment:
classes not added

Reply:
you need to add all 5 classes for a different count of stars,

Reply:
the first time I did not understand, what did you say. 
thank you for the fix